include/multiverso/net/mpi_net.h

```diff
@@ -40,7 +47,7 @@ namespace {
     /* TODO: Windows */
   #elif defined(__APPLE__)
     /* TODO: Mac OS X */
-  #else
+  #elif defined(__linux__)
```

Comment:
I think `else` here try to support not only linux but also other platforms.
Is there any reason this must be done?